client-data/tools/cursor/cursor.js

```diff
@@ -106,7 +106,8 @@
     function draw(message) {
         var cursor = getCursor("cursor-" + (message.socket || 'me'));
         cursor.style.transform = "translate(" + message.x + "px, " + message.y + "px)";
+        if (Tools.isIE) cursor.setAttributeNS(null, "transform", "translate(" + message.x + " " + message.y + ")");
```

Comment:
Is there a reason we don't use the circle position in all cases ?

Reply:
You mean the `x` and `y` property? 
There was a performance benefit to using css transforms as it avoids content reflow and allows gpu acceleration.